OrbitClientModel/CaptureData.cpp

```diff
@@ -112,6 +112,22 @@ const std::string& CaptureData::GetFunctionNameByAddress(uint64_t absolute_addre
   return function_name;
 }
 
+// Find the start address of the function this address falls inside. Use the Function returned by
+// FindFunctionByAddress, and when this fails (e.g., the module containing the function has not
+// been loaded) use (for now) the LinuxAddressInfo that is collected for every address in a
+// callstack.
+uint64_t CaptureData::GetFunctionBaseAddressByAddress(uint64_t absolute_address) const {
```

Comment:
This returns absolute_address if function was not found. I am not sure this is intuitive. Maybe use std::optional or ErrorMessageOr<> here? and let the caller decide what the value should be in this case using value_or() for example.

Reply:
Maybe FindFunctionAbsoluteAddressByAddress?

Reply:
Done